sdk/identity/azure-identity/README.md

```diff
@@ -137,6 +130,16 @@ public void createDefaultAzureCredentialForIntelliJ() {
 }
 ```
 
+## Managed Identity Support
+The [Managed identity authentication](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview) is supported via `ManagedIdentityCredential` for following Azure Services:
```

Comment:
Should this mention that managed identity support is supported via either the `DefaultAzureCredential` or the `ManagedIdentityCredential` directly?

Reply:
Yep although, we've covered the DAC flow above with Managed Identity example with DAC too.
But if someone directly jumps here, it would be helpful for them to read of the support via both credentials. Adding it.